brands/shop/furniture.json

```
@@ -819,6 +819,18 @@
         "shop": "furniture"
       }
     },
+    {
+      "displayName": "Sotka",
+      "id": "sotka-32ceba",
+      "locationSet": {"include": ["fi"]},
+      "tags": {
+        "brand": "Sotka",
+        "brand:wikidata": "Q18689173",
+        "brand:wikipedia": "fi:Sotka",
```

looks like it should be:
```suggestion
        "brand:wikipedia": "fi:Sotka (ketju)",
```

---

Sorry about it! updated it! Please review thanks!